### ISSUE TITLE
Response length returned exceeded our limit for reading for request

### ISSUE THREAD

**Issue by HiradRoshandel** [scoutapp/scout-apm-php#289]
Hi @asgrim, 

We deployed scout APM into our Production laravel application but we continuously see the following error in the logs for every request that is hitting the API:

```
[2022-09-26 17:52:40] production.NOTICE: [Scout] Response length returned (1667509603) exceeded our limit for reading (10000000) for Scoutapm\Events\Request\Request. Address was: tcp://127.0.0.1:6590
```

Is this a bug? or how can we fix the problem?

Thanks

**Comment by asgrim** [comment]
@aconital something definitely seems off there; there should be some "debug" level output just before that, something like:

`Sending metrics from %d collected spans` - if you have debug level logging, does that number seem particularly high? I'm concerned as we already put a limitin place to limit the number of spans. That response length is definitely off IMO.

Emailing Scout support (support@scoutapm.com) may also have some information on this if it is a known issue.

**Comment by asgrim** [comment]
This is fixed in #290 and released in [8.6.2](https://github.com/scoutapp/scout-apm-php/releases/tag/8.6.2).
